portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -84,7 +84,7 @@
 				languageValue = LocalizationUtil.getLocalization(xml, curLanguageId, false);
 			}
 
-			if (Validator.isNotNull(languageValue) || (!ignoreRequestValue && (request.getParameter(name + StringPool.UNDERLINE + curLanguageId) != null))) {
+			if (Validator.isNotNull(languageValue) || (!ignoreRequestValue && Validator.isNotNull(ParamUtil.getString(request, name + StringPool.UNDERLINE + curLanguageId, languageValue)))) {
```

Comment:
@veroglez, it kinda makes sense to me, but it's hard to tell, it's been a long time since I've worked on this...

What are you expecting this would do? If you explain it maybe it's easier for me or someone else to make sure it's correct.

Reply:
Currently what is happening is that when the form is submitted and the request fails (or it is not saved in the backend) the values of the input localized are obtained from the request.

In this case, even when the sent value is an empty string the input localized is adding the name_es_ES to the request, resulting in the translated language. This is conceptually a mistake, since an empty string should not be evaluated.

With this change, it is intended that this doesn’t happen and the value is only evaluated if the sent string is not empty. In this way, the language will only be saved as translated if it has a value that is not empty.